src/Microsoft.FeatureManagement/FeatureFilters/Cron/CronField.cs

```diff
@@ -0,0 +1,241 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+using System.Collections;
+
+namespace Microsoft.FeatureManagement.FeatureFilters.Cron
+{
+    /// <summary>
+    /// The CronField uses a BitArray to record which values are included in the field.
+    /// </summary>
+    internal class CronField
+    {
+        /// <summary>
+        /// Whether the CronField is an asterisk.
+        /// </summary>
+        public bool MatchAll { get; private set; } = false;
```

Comment:
I know I suggested this name, but I think `MatchesAll` would be better. Also initializing boolean to `false` is unnecessary.


```suggestion
        public bool MatchesAll { get; private set; }
```

Reply:
Also, please put properties after constructor.